internal/gps/strip_vendor.go

```diff
@@ -16,29 +16,27 @@ func stripVendor(path string, info os.FileInfo, err error) error {
 		return err
 	}
 
-	if info.Name() == "vendor" {
-		if _, err := os.Lstat(path); err != nil {
```

Comment:
why is it OK to drop this lstat?

Reply:
It was pretty much not doing anything. `lstat` has the same error responses as `stat`. If we already have a `FileInfo` then `stat` has succeeded.

Anyhow, I can restore it if we are unsure and don't want to break anything.

cc: @spenczar Any idea what's the original intent of this?

Reply:
@ibrasho Just checking in to confirm that the original purpose of this got lost after a few rewrites of the surrounding code (including my own!). Your fix here looks correct to me. 👍